lib/index.js

```diff
@@ -126,18 +155,20 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
 
   // Expose ramenSettings so Ramen.go() can see it
   window.ramenSettings = ramenSettings;
 
   // Ramen.go() will figure things out if called multiple times
   window.Ramen.go();
+
+  this.identified = true;
 };
 
 Ramen.prototype.group = function(group) {
-  // If there is not already a ramenSettings object, or if Ramen isn't loaded,
-  // then do nothing here.
-  if (!(window.ramenSettings && window.Ramen)) {
+  // If there has not been an identify call yet, do nothing.
+  if (!this.identified) {
```

Comment:
Ok but if localstorage is fair game, then the following can happen:

1. Page loads
2. `#group` is called
3. `this.analytics.user().email()` and `this.analytics.user().id()` exist thanks to localstorage

At this point, the Ramen `group` call should still return immediately because `#identify` has not been called yet, and therefore `ramenSettings` hasn't be initialized correctly.

Now, I look at that and think "Well gee, Ryan, since you have `email()` thanks to localstorage, you can still setup `ramenSettings` -- you just need to factor out some of the logic in your current `identify`"

I'm more than willing to make that change so that `group` (and the other methods for that matter: `track` and `page`) can work without `identify` being called if we do have `email()` and `id()` thanks to localstorage, but that's a pretty big refactor. If I did that, would you have time to review and merge it in before the announcement goes out tomorrow?